src/utils/hashPrefixTester.js

```diff
@@ -0,0 +1,10 @@
+const config = require('../config.js');
+
+const minimumPrefix = config.minimumPrefix || '3';
+const maximumPrefix = config.maximumPrefix || '32'; // Half the hash.
```

Comment:
Is there a reason to have a maximum?

Reply:
probably not but it doesn't hurt

Reply:
Having a maximum is more to limit clients from doing the wrong thing. Discourage sending too much of the hash, as that would just be tossing the anonimity layer out again.